src/dev/flang/parser/Parser.java

```diff
@@ -3389,11 +3389,8 @@ List<Cond> invariant(boolean atMinIndent)
   /**
    * Parse implRout
    *
-implRout    : "is" "abstract"
-            | ARROW "abstract"
-            | "is" "intrinsic"
+implRout    : ARROW "abstract"
```

Comment:
I think we should leave the grammar as is. Parser does parse e.g. `is abstract` even though it then emits an error.

Reply:
I think we shoud do both: change the grammar (which becomes simpler) and change the parser to not accept `is abstract`, etc. Then, we might not need new error messages and everything becomes easier...